docs/jobs-queue/queues.mdx

```diff
@@ -25,7 +25,32 @@ Then, you could configure two different runner strategies:
 
 ## Executing jobs
 
-As mentioned above, you can queue jobs, but the jobs won't run unless a worker picks up your jobs and runs them. This can be done in two ways:
+As mentioned above, you can queue jobs, but the jobs won't run unless a worker picks up your jobs and runs them. This can be done in three ways:
+
+#### Cron jobs
+
+You can use the jobs.autoRun property to configure cron jobs:
+
+```ts
+export default buildConfig({
+  // Other configurations...
+  jobs: {
+    tasks: [
+      // your tasks here
+    ],
+    autoRun: (payload) => [
+      {
+        default: '*/5 * * * *', // every hour at minute 0
+        limit: 100, // limit of retries
```

Comment:
Instead of  `// limt of retries`, I'd say `// limit jobs to process each run` or similar, retries is configured on the job.